controllers/managedmcg_controller_test.go

```diff
@@ -72,6 +83,17 @@ func newManagedMCGFake() mcgv1alpha1.ManagedMCG {
 	return managedMCGFake
 }
 
+func newConsoleDeploymentFake() appsv1.Deployment {
+	ConsoleDepFake := appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "mcg-ms-console",
+			Namespace: "openshift-storage",
```

Comment:
`openshift-storage`? Do we want this to fail? If we are trying to cover Vinayak's PR, I'd suggest doing so for the happy path first, then the corner cases. LMK WDYT.

Reply:
Why are you thinking it should not be "openshift-storage"? I am targeting happy path only...May be I am missing something.

By the way, I am not very good in short forms :) :)  "LMK WDYT"?? I spent more time in finding its meaning :)

Reply:
I'm assuming deployment should ideally be under `redhat-data-federation`? Is that not the case?

Reply:
"redhat-data-federation" we use in E2E testing.
I understand the confusion :)